docs/src/views/AccessibilityBehaviors.tsx

```diff
@@ -0,0 +1,256 @@
+import * as React from 'react'
+import { Header } from '@stardust-ui/react'
+import DocPage from '../components/DocPage'
+import CodeSnippet from '../components/CodeSnippet'
+import { code, link } from '../utils/helpers'
+
+export default () => (
+  <DocPage title="Accessibility Behaviors">
+    <Header as="h2">Content</Header>
+    <ul>
+      <li>{link('Overview', '/accessibility-behaviors#overview')}</li>
+      <li>{link('ARIA attributes', '/accessibility-behaviors#aria-attributes')}</li>
+      <li>{link('Overriding behaviors', '/accessibility-behaviors#overriding-behaviors')}</li>
+    </ul>
+    <Header as="h2">Overview</Header>
+    <p>
+      In Stardust, accessibility behaviors encapsulate keyboard and screen reader navigation. They
```

Comment:
... encapsulate logic required for keyboard navigation, focus handling and screen reading.